CODEOWNERS

```diff
@@ -99,7 +99,7 @@ extensions/filters/common/original_src @klarose @mattklein123
 # http inspector
 /*/extensions/filters/listener/http_inspector @yxue @lizan
```

Comment:
@envoyproxy/envoy-maintainers we need new owners for the extensions that Lizan was the only maintainer.

Reply:
I can help for http_inspector. I have some related experience and we rarely update this. It won't bring much burdens.

Reply:
@lizan if you are going to be around we'd love for you to continue to own extensions.  Otherwise you can change to @UNKNOWNED for now and we will either replacements or bump to contrib later on

Reply:
If needed, I am also happy to take transport_socket related extensions and the grpc_stats one.

Reply:
> I can help for http_inspector. I have some related experience and we rarely update this. It won't bring much burdens.

http_inspector directly uses third party http-parser to parse of http protocol detection. It is soon going to be deprecated in favor of Balsa. That change is due in near future.

Reply:
I added a few UNOWNED and add back myself to extension owners for some of the topics. While we still need at least one sponsor maintainer for those extensions.